src/test/java/org/opencds/cqf/tooling/utilities/ECQMCreatorIT.java

```diff
@@ -67,1278 +68,427 @@ private String measureToString(Measure measure) {
         return measureResourceContent;
     }
 
-    @Test
-    public void TestCMS125FHIR() {
-        // Extract the bundle
-        // NOTE: This is a 2021-AUFHIR measure, this is the test to use as the template to add the rest of the content for testing
+    private  List<DataRequirement> StartMatOutputTest(String matBundleName, String measureLibraryName){
```

Comment:
had to more some common functionality into a method to make it so I could save the refreshed measure to a directory